pkg/tessera/safeint.go

```diff
@@ -20,13 +20,13 @@ import (
 	"math"
 )
 
-// safeInt64 holds equivalent int64 and uint64 integers.
-type safeInt64 struct {
-	u uint64
-	i int64
+// SafeInt64 holds equivalent int64 and uint64 integers.
+type SafeInt64 struct {
+	U uint64
```

Comment:
should these still be un-exported with getters (`U()`, `I()`)? I worry someone would just set them and pass SafeInt64 around externally, without using NewSafeInt64?